docker-compose.yml

```diff
@@ -14,7 +14,11 @@ x-app: &x-app
 services:
   react-ui-wc:
     <<: *x-app
-    command: yarn start:wc
+    command: yarn start
     ports:
       - "3011:3011"
     container_name: react-ui-wc
+networks:
+  default:
+    name: shared-development
+    external: true
```

Comment:
I think we might not need this anymore? Or did you add it back in because it turns out we do?